packages/fx-core/templates/plugins/resource/apim/README.md

```diff
@@ -1,6 +1,6 @@
 # Publish Teams Backend (Azure Functions) to Azure API Management
 
-Azure API Management (APIM) is used to create consistent and modern API gateways for existing backend services. With Teams Toolkit or TeamsFx CLI, you can easily publish your backend APIs (Azure Functions) to existing or new APIM instance.
+Azure API Management (APIM) is used to create consistent and modern API gateways for existing backend services. With Teams Toolkit or TeamsFx CLI, you can easily publish your backend APIs (Azure Functions) to APIM instance.
 
 ## Prerequisite
 
```

Comment:
line#8 
```An M365 account, if you do not have an M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)```
should be `apply for one from xxx`

Reply:
line10 
```
Ensure the resource provider 'Microsoft.ApiManagement' is registered for the subscription by following [this document]
```
should be `by following [document](xxx)`. Please help remove the `this`.

Reply:
Thanks!